tests/faultydata/variants/duplicated_sampleid/README.md

```diff
@@ -0,0 +1,6 @@
+Data file example_1.vcf.gz was copied from tests/data/variants/example_1, then change sample id S2 to S1
+
+Index file was generated by
+```bash
```

Comment:
bash is a shell, and you're not using any bash-specific utilities here, so the mention of bash is superfluous.

I would format it as just:

```
$ tabix example_1.vcf.gz
```

The dollar sign indicating a command prompt